Examples/CodePushDemoAppCpp/app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "CodePushDemoAppCpp",
+  "displayName": "CodePushDemoAppCpp"
+}
```

Comment:
Could you please fix `No newline at the end of file` warning in all files?

Reply:
I have added newlines at the end of each file. Hopefully the warnings are fixed now.